tests/unit/sagemaker/config/test_config_schema.py

```diff
@@ -199,3 +199,13 @@ def test_invalid_s3uri_schema(base_config_with_schema):
     config["SageMaker"] = {"PythonSDK": {"Modules": {"RemoteFunction": {"S3RootUri": "bad_regex"}}}}
     with pytest.raises(exceptions.ValidationError):
         validate(config, SAGEMAKER_PYTHON_SDK_CONFIG_SCHEMA)
+
```

Comment:
would be great to also include this new parameter in tests/data/config/config.yaml, and in test_valid_endpoint_config_schema